### PR TITLE
Simplify sponsor page

### PR DESCRIPTION
**Pull request by vsmart**
As listed here: https://github.com/clojurebridge-berlin/clojurebridge-berlin.github.io/issues/30

the intention is to simplify the page and more clearly list what each level of payment gets you 

I also tried to remove the association that all Gold/Platinum _must_ be direct sponsorships - does that make sense? In my understanding those tiers are primarily about the amout of $ not about the //direct sponsorship// - and I tried to clarify that by changing the "you pay for...." to "you may for example (e.g.) pay for lunch". That way we're not tied to having e.g. only 1 gold sponsors and can divide the money how it makes sense to us. 

Plus, I tried to remove the word "tier" cause I think it confuses Germans 😹

### CODE REVIEW

**Comment by vsmart** [comment]
@plexus @socksy @Realtin et al please have a look 👀

**Comment by socksy** [comment]
Some great changes! Thank you. But I won't lie that I was secretly hoping German companies would be confused by being called "Golden Tier"s and so on.🐈

I am a big fan of removing all the text and putting things in bullet points/lists/tables.

A few points:
- Are the bullet lists final? If not, disregard my next points and questions
- It's difficult to see what the differences are between gold and platinum tiers. Is it just the higher placement thing? We may want to brainstorm something more than just "higher placement"
- Maybe we should also point out that platinum is 600**_+_**...
- What do the "e.g."s mean? I'm guessing it's what they're sponsoring for, but does this make sense when they're not doing direct sponsorship? 
- Having "swag for attendees" twice, once as what they're sponsoring (I think) and one for as a benefit they could have, really confused me. Perhaps an example of "providing ClojureBridge branded T-Shirts to the attendees" (but I imagine that would be more platinum tier?)

Also, I don't get this change entirely:

> If you've opted for a direct sponsorship **(only for Platinum or Gold sponsors)**, then your transaction will be with our suppliers directly.

I'm guessing I'm confused in its relation to this optional direct sponsorship thing. This kinda implies that Platinum and Gold sponsors are only direct sponsors, which as you say, they're not (the main reasoning for implying it before was to see if we could save some money on the big sponsors, as we wouldn't be charged by travis for that usage). Also, it suggests that we won't take direct sponsors for less than the gold sponsor amount (perhaps we wouldn't, but I guess it depends on the stress and the amount/thing).

File: sponsor_us.md [hunk 1]
```diff
@@ -17,35 +17,32 @@ By supporting us you help to make the world a little better. Your company also
...
 
-All our sponsors are listed on our website and on our event page on
-ClojureBridge.org. They'll receive a shoutout at the start and end of the event
-along with their logos in our presentations. All our sponsors will also have
-the opportunity to offer "swag" (stickers, note books, thumb drives, etc.) at
-the event.
+### Gold Sponsor - 500€
+* e.g. swag for the attendees
+* Your logo on our website
+* Shoutout on our social media pages
```
Comment (socksy) [inline sponsor_us.md]: TODO: remove

**Comment by socksy** [comment]
Looks great to me

**Comment by Malwine** [comment]
@vsmart 
giphy7

**Pull request merged**